### DISCUSSION TITLE
Add descriptions to "Requests", "Pull Payments" and "Payouts"

### DISCUSSION THREAD

**Discussion by BenGWeeks** [btcpayserver/btcpayserver#3770] [Ideas / Feature Requests]
Hi there,

As a user, I'm not entirely sure what these were for. As a shop owner, for example, requesting funds from the Admin, or an end-user, or as an Admin, am I requesting funds from the shop owner, or an end-user?

I would suggest something like:

**Requests**

Payment Requests are a feature which allows BTCPay store owners to create long-lived invoices. Funds paid to a payment request use the exchange rate at the time of payment. This allows users to make payments at their convenience without having to negotiate or verify exchange rates with the store owner at the time of payment.

Users can pay requests in partial payments. The payment request will remain valid until it is paid in full or if the store owner requires an expiration time. Addresses are never reused. A new address is generated each time the user clicks pay to create an invoice for the payment request.

Store owners can also print payment requests (or export invoice data) for record-keeping and accounting. BTCPay automatically labels invoices as Payment Requests in your store's invoice list.

For more information on payment requests, please see https://docs.btcpayserver.org/PaymentRequests/

**Pull Payments**

Pull Payments allow for a receiver to request funds ("pull") as apposed to the sender knowing an address to "push" a payment to. This can be useful for sending refunds to purchasers, or paying suppliers for example.

For more information on pull payments, please see https://docs.btcpayserver.org/PullPayments/.

**Payouts**

The payouts view gives an overview of the current pull payments and their status. If, for example, a refund has been issued and the claimant has accepted, this will show in Payouts. Here you will find the options to Approve and directly send the requested amount of the refund. When there are multiple instances of Pull payments, these can be selected and batched for sending out at once. In a future version, we expect this to have scheduling options.

For more information on payouts, please see https://docs.btcpayserver.org/Walkthrough/#payouts

Many thanks.

Ben

**Comment by dstrukt** [comment]
We've talked about adding basic descriptors to many of these views in the past, so definitely agree to help alleviate some of this confusion, will reference these write-ups in the process of adding those descriptors.

Cheers!

**Comment by dstrukt** [comment]
Updates on this:
https://github.com/btcpayserver/btcpayserver/issues/3266
